server/Logic/Order.ts

```diff
@@ -100,7 +100,16 @@ export class Order implements IOrder {
 		return this.changeOrderStatus('assigned', true, true);
 	}
 
-	giveFeedback(_review: string, _score: number): boolean {
-		throw new Error('Method not implemented');
+	async giveFeedback(
+		review: string,
+		score: number
+	): Promise<ResponseMsg<void>> {
+		if (score < 1 || score > 5) {
+			return makeFail('Review score must be between 1 and 5.');
+		}
+		this.orderDAO.review.content = review;
```

Comment:
Can you uncomment (remove the underscore from the file name) the tests for reviews?